### ISSUE TITLE
[PV3.4] WikiPathways plugin doesn't load remote pathways

### ISSUE THREAD

**Issue by egonw** [PathVisio/pathvisio#214]
Steps:

* Menu Plugins -> WikiPathways -> Search

Then I get a dialog follows by a second dialog with the error message "Unable to connect to WikiPathways":

![image](https://github.com/user-attachments/assets/d420be4e-13a2-4adf-8215-c97d345758fa)

In my `PathVisio.log` I see this line show up when doing the above steps and closing the dialogs:

```
Error: Unable to conntect to WikiPathways
java.lang.NoClassDefFoundError: org/jdom/input/SAXBuilder
```

**Comment by traybug23** [comment]
Hi @egonw 
I am interested in working on a fix for this issue, is their any additional information I should know of? And, is there any reccomended resources to look into before starting on the fix?

**Comment by egonw** [comment]
hi @traybug23, I think I need to close this issue. It is a bug relating a system that will go offline very soon. Thank you for your offer, though.

**Comment by traybug23** [comment]
Oh, thank you for letting me know. Was this JDOM library loading issue fixed in the newer version of the system? (I might be mistaken but is it the pathvisio4-ant version?) If so, then out of curiosity I'd love to understand how it was resolved to learn how library lodading issues are handled.

**Comment by egonw** [comment]
No, nothing to do with the code. The system that is is connecting will go offline.
